src/devices/sonoff.ts

```diff
@@ -950,13 +950,21 @@ const definitions: DefinitionWithExtend[] = [
         exposes: [e.cover_position(), e.battery()],
     },
     {
-        zigbeeModel: ['Z111PL0H-1JX', 'SA-029-1', 'SA-028-1'],
+        zigbeeModel: ['SA-029-1', 'SA-028-1'],
         model: 'SA-028/SA-029',
         vendor: 'SONOFF',
         whiteLabel: [{vendor: 'Woolley', model: 'SA-029-1'}],
         description: 'Smart Plug',
         extend: [onOff(), forcePowerSource({powerSource: 'Mains (single phase)'})],
     },
+    {
+        zigbeeModel: ['Z111PL0H-1JX'],
+        model: 'SA-028/SA-029',
```

Comment:
```suggestion
        model: 'Z111PL0H-1JX',
```

In that case make this change, a `model` can not be re-used (that's why the tests are failing)